server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -856,11 +858,46 @@ int getAllTimeseriesCount(PartialPath prefixPath) throws MetadataException {
     if (nodes.length == 0 || !nodes[0].equals(root.getName())) {
       throw new IllegalPathException(prefixPath.getFullPath());
     }
+    Holder<Integer> countHolder = new Holder<>(0);
     try {
-      return getCount(root, nodes, 1);
+      getAllTimeseriesCountHelper(root, nodes, 1, countHolder, false);
```

Comment:
I feel the method name `getAllTimeseriesCountHelper` is not clear enough. How about just keep the origin method name `getCount`?

Reply:
ok,